core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -473,10 +474,11 @@ object ConfigCommand extends Config {
   }
 
   private def describeClientQuotasConfig(adminClient: Admin, entityTypes: List[String], entityNames: List[String]) = {
+    warn(s"entityNames = $entityNames")
     getAllClientQuotasConfigs(adminClient, entityTypes, entityNames).foreach { case (entity, entries) =>
       val entityEntries = entity.entries.asScala
-      val entityStr = (entityEntries.get(ClientQuotaEntity.USER).map(u => s"user-principal '${u}'") ++
-        entityEntries.get(ClientQuotaEntity.CLIENT_ID).map(c => s"client-id '${c}'")).mkString(", ")
+      val entityStr = (entityEntries.get(ClientQuotaEntity.USER).map(u => s"user-principal '${Option(u).filterNot(_.isEmpty).getOrElse("default")}'") ++
```

Comment:
this is clever, but I'd really rather have a simple `if (u.isEmpty) "default" else u`